openpype/tools/utils/lib.py

```diff
@@ -126,18 +122,42 @@ def get_qta_icon_by_name_and_color(icon_name, icon_color):
     return icon
 
 
+def get_project_icon(project_doc):
+    if project_doc:
+        icon_name = project_doc.get("data", {}).get("icon")
+        icon = get_qta_icon_by_name_and_color(icon_name, "white")
+        if icon:
+            return icon
+
+    return get_qta_icon_by_name_and_color(
+        "fa.map", get_default_entity_icon_color()
+    )
+
+
+def get_asset_icon_name(asset_doc, has_children=True):
+    if asset_doc:
```

Comment:
Why does this need an `if asset_doc`? To me it'd feel very weird for the function to succeed if I don't pass it a valid document? Shouldn't that just error?